.github/workflows/tests.yml

```diff
@@ -13,11 +13,14 @@ jobs:
         plone:
           - "5.2.5"
           - "5.2.6"
```

Comment:
I would suggest using 5.2-latest and 6.0-latest as it would be easier to check with newer releases (fyi: Plone is already on 6.0.0.2)

Reply:
Using "-latest" is still an anti-pattern (if not THE anti-pattern) of CI IMHO. :)

What about 6.0.0.2 anyways? Since when are we using four-version numbering?